go.mod

```diff
@@ -88,7 +88,7 @@ require (
 	github.com/hashicorp/hcl v1.0.0 // indirect
 	github.com/huandu/xstrings v1.3.3 // indirect
 	github.com/imdario/mergo v0.3.13 // indirect
-	github.com/inconshreveable/mousetrap v1.0.1 // indirect
+	github.com/inconshreveable/mousetrap v1.1.0 // indirect
```

Comment:
why is this getting updated?

Reply:
@mkumatag It got automatically updated in my local it seems, It works even without these changes, have removed the changes and pushed again. Please review